CMakeLists.txt

```diff
@@ -1,7 +1,19 @@
 cmake_minimum_required (VERSION 2.8.11)
+cmake_policy(SET CMP0048 NEW)
```

Comment:
Policy introduced in 3.0: https://cmake.org/cmake/help/latest/policy/CMP0048.html
But minimum version set to 2.8. If user will try to run with CMake 2.8 he will hit error.

Reply:
Done.

Reply:
?